app/src/main/java/com/android/voyageur/ui/trip/assistant/Assistant.kt

```diff
@@ -264,14 +335,17 @@ fun SettingsDialog(
           Spacer(modifier = Modifier.height(16.dp))
 
           Row(verticalAlignment = Alignment.CenterVertically) {
+            // Switch to provide final activities
             Text(stringResource(R.string.provide_final_activities), modifier = Modifier.weight(1f))
             Switch(
                 checked = provideDraftActivities,
                 onCheckedChange = onProvideFinalActivitiesChanged,
                 modifier = Modifier.testTag("provideFinalActivitiesSwitch"))
           }
           Spacer(modifier = Modifier.height(4.dp))
+
           Row(verticalAlignment = Alignment.CenterVertically) {
+            // Switch to use only the current user's interests
```

Comment:
Here in the dialog the text is very close to the button ("interests" is almost touching the button). Maybe we can change it to "your interests" / "your user's interests" so it's a bit shorter.

Reply:
Thanks for pointing this out! I added a spacer between the text and the switch